tools/wptrunner/wptrunner/metadata.py

```diff
@@ -372,8 +376,8 @@ def update_from_log(self, log_file):
             success = self.get_wptreport_data(line)
             if not success:
                 break
-        else:
-            return
+            else:
```

Comment:
Yikes, this looks scary! This looks like it would change the semantics here to only work for single-line encapsulated data (which is what line 363 appears to be trying to handle already). What leads to this change?

Reply:
Oops! This was my misunderstanding. Reverted.